### ISSUE TITLE
WCAG 4.1.2 test 21 false negative

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#78]
[This very simple PDF document](https://github.com/veraPDF/veraPDF-wcag-algs/files/6970249/test-document-13.pdf) contains two fake lists that are tagged as normal paragraphs. The first one is a fake unordered list with items delimited by dashes ('-'), the second is a fake ordered list with items delimited by numbers.

This document should generate error **WCAG 4.1.2 test 21 "Missing List element(s)"** for each one of the two fake lists, but it doesn't, so this is a false negative.

**Comment by jzuidweg** [comment]
This issue has been tested and is solved. It can be closed.
